src/main/java/cms/gov/madie/measure/resources/TestCaseController.java

```diff
@@ -222,27 +248,63 @@ public ResponseEntity<Map<String, Object>> shiftQdmTestCaseDates(
       }
       unlocked.add(testCaseId);
     }
+    return unlocked;
+  }
 
-    List<String> shiftedIds =
-        qdmTestCaseShiftDatesService.shiftTestCaseDates(
-            measureId, unlocked, shifted, accessToken, principal);
+  private Map<String, Object> populateShiftedAndFailed(
+      List<TestCase> allTestCases, List<String> shiftedIds, List<String> failedIds) {
+    List<String> shiftedTestCases =
+        new ArrayList<>(
+            allTestCases.stream()
+                .filter(testCase -> shiftedIds.contains(testCase.getId()))
+                .map(
+                    testCase ->
+                        StringUtils.isBlank(testCase.getSeries())
+                            ? testCase.getTitle()
+                            : testCase.getSeries() + " - " + testCase.getTitle())
+                .toList());
 
-    Map<String, Object> response = new HashMap<>();
-    response.put("shifted", shiftedIds);
-    response.put("failed", failed);
+    List<String> failedTestCases =
+        new ArrayList<>(
+            allTestCases.stream()
+                .filter(testCase -> failedIds.contains(testCase.getId()))
+                .map(
+                    testCase ->
+                        StringUtils.isBlank(testCase.getSeries())
+                            ? testCase.getTitle()
+                            : testCase.getSeries() + " - " + testCase.getTitle())
+                .toList());
 
-    return ResponseEntity.ok(response);
+    Map<String, Object> response = new HashMap<>();
+    response.put("shifted", shiftedTestCases);
+    response.put("failed", failedTestCases);
+    log.info("shift dates qdm shiftedTestCases: " + shiftedTestCases.toString());
+    log.info("shift dates qdm failedTestCases: " + failedTestCases.toString());
```

Comment:
Parameterize log statements and remove unnecessary `toString`.
```suggestion
    log.info("shift dates qdm shiftedTestCases: {}", shiftedTestCases);
    log.info("shift dates qdm failedTestCases: {}", failedTestCases);
```

Reply:
@jkotanchik-SB done, thanks again for reminder!